### PR TITLE
feat!: types

### PR DESCRIPTION
**Pull request by reslear**
closes: #3 

___

Screenshot 2024-01-12 at 00 55 03
Screenshot 2024-01-12 at 00 53 51

### CODE REVIEW

File: src/App.vue
Comment (niklasfjeldberg) [inline src/App.vue:1]: I am not sure I understand the benefits of this approach. Could you give some insight? Are you trying to show how you would use the block with external data that could be updated?

File: lib/text.ts
Comment (niklasfjeldberg) [inline lib/text.ts:1]: I guess this is a better temporary fix then how it currently is, but it does not solve issue https://github.com/niklasfjeldberg/vue-strapi-blocks-renderer/issues/4

File: lib/index.ts
Comment (niklasfjeldberg) [inline lib/index.ts:1]: What exactly does this do, make all types available?

Comment (reslear) [reply lib/index.ts:1]: no changes, only for generate correct `vite-plugin-dts` types

File: lib/text.ts
Comment (reslear) [reply lib/text.ts:1]: yes, better to generate with `vue-tsc`, but first we need to separate the package from the playground.

File: src/App.vue
Comment (reslear) [reply src/App.vue:1]: it's just `as` not correct to use in the template

File: lib/text.ts
Comment (reslear) [reply lib/text.ts:1]: Yes, you're right. I got it wrong. It doesn't solve #4

**Comment by reslear** [comment]
I also made the patch locally and it's already working in production like a charm :)

**Pull request merged**